_data/projects/AlaSQL.yml

```diff
@@ -0,0 +1,22 @@
+ 
+name: AlaSQL
+desc: Javascript database for traditional SQL queries. Smaller and faster alternative to sql.js   
+site: https://github.com/agershun/alasql
+tags:
+- Meteor
+- Good vibes 
```

Comment:
We've got *a lot* of tags here. Can we drop this and just focus on the technical stuff for now?